formwidgets/mltextarea/partials/_mltextarea.htm

```diff
@@ -15,7 +15,8 @@
             placeholder="<?= e(trans($field->placeholder)) ?>"
             class="form-control field-textarea size-<?= $field->size ?>"
             autocomplete="off"
-            <?= $field->getAttributes() ?>><?= e($field->value) ?></textarea>
+            <?= $field->getAttributes() ?>
+            <?= $field->required ? 'required' : '' ?>><?= e($field->value) ?></textarea>
```

Comment:
```suggestion
            <?= $field->required ? 'required' : '' ?>>
            <?= e($field->value) ?>
        </textarea>
```

Reply:
Literally anything (including line breaks and white space) between the `<textarea></textarea>` tags will be added to the rendered element. This is not what you want.

Reply:
You're right, I had a brain cramp while focusing on coding style... :)